server/middlewares/auth.js

```diff
@@ -0,0 +1,29 @@
+import jwt from "jsonwebtoken";
+
+const verifyToken = (req, res, next) => {
+    const bearerHeader = req.headers['authorization'];
+    
+    if (typeof bearerHeader !== 'undefined') {
+        const bearer = bearerHeader.split(' ');
+        const bearerToken = bearer[1];
+        req.token = bearerToken;
+        jwt.verify(req.token, 'secretKey' , (err) => {
```

Comment:
Please use an actual secret key and make sure it is added in the .env file for security purposes.